src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -10428,7 +10428,7 @@ void func_80848C74(PlayState* play, Player* this) {
 
 void func_80848EF8(Player* this) {
     if (CHECK_QUEST_ITEM(QUEST_STONE_OF_AGONY)) {
-        f32 temp = 200000.0f - (this->unk_6A4 * 5.0f);
+        f32 temp = 200000.0f - (this->closestSecretDistSq * 5.0f);
```

Comment:
```suggestion
        f32 temp = (SQ(200.0f) * 5.0f) - (this->closestSecretDistSq * 5.0f);
```
Perhaps? And the `4000000.0f` in this function could be `SQ(2000.0f)`
Might be worth naming `unk_6A0` and `func_80848EF8` since both are only used here with `closestSecretDistSq`, so might be good to document this as a group.

Maybe `func_80848EF8` -> `Player_RumbleNearSecret`
`unk_6A0` -> `secretRumbleTrigger`
Just brainstorming?

Not essential that these are named now though